content/articles/domains-uk.markdown

```diff
@@ -122,6 +122,24 @@ You can now decide to [point the domain to DNSimple name servers](/articles/dele
 
 To transfer a `.UK` domain away from DNSimple, the procedure is similar to the [instructions to transfer in](#transfer), but you will need to get the IPS tag of the gaining registrar.
 
+<div class="section-steps" markdown="1">
+#### Initiating the transfer
+
+1.  Go to the registration section and click <label>Transfer Out</label>.
+
+    ![Transfer out link](/files/domain-transfer-out-action.png)
+
+#### Approving the transfer
+
+1. Enter the IPS tag of the gaining registrar. **This is case-sensitive, so make sure the IPS TAG is in all capital letters.**
+
+    ![IPS Tag](/files/uk-domain-transfer-out-ips-tag.png)
+
+1. Click <label>Request Transfer Out</label>.
+
+    This will unlock the domain and submit the request to the registry within 24 hours.
```

Comment:
For .UK this is not correct. It's not really a request, the domain will immediately be pushed to the new IPS tag owner.